src/content/vintage/getting-started/observability/logging/architecture/index.md

```diff
@@ -0,0 +1,72 @@
+---
+linkTitle: Logging architecture
+title: Logging architecture
+description: Documentation on the logging architecture deployed and maintained by Giant Swarm.
+weight: 80
+menu:
+  main:
+    identifier: getting-started-observability-logging-architecture
+    parent: getting-started-observability-logging
+user_questions:
+  - What is the logging architecture?
+  - Why is Giant Swarm using Loki?
+  - Why is Giant Swarm recommending Loki?
+  - Which logs are stored by Giant Swarm?
+  - Where are the logs stored by Giant Swarm?
+aliases:
+  - /getting-started/observability/logging/architecture
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2024-03-21
+---
+
+Logging is an important pillar of observability and it is thus only natural that Giant Swarm provides and manages a logging solution for operational purposes.
+
+This document gives an overview of how logging is managed by Giant Swarm, including which logs are stored, which tools we use to ship and store them, as well as why we chose those tools in the first place.
```

Comment:
This doesn't tell the customer much why it's helpful for them. What are operational purposes? Does that mean they or only we can make use of it? I'd recommend shortening this intro, include customer-relevant points and only describe what's important to them. I assume that customers can make use of the MC Grafana instance (see existing article `How to access Grafana` which btw has a misspelled title `it's`/`its`) and therefore we should strongly hint here why the architecture is important for customers (e.g. long-term storage, persistence, easy access via Grafana UI, ...) but not go into extensive detail or opinions ("the full Elastic stack is obviously out of the question").

@pipo02mix I'm very unclear what was discussed around vintage docs. Shouldn't we start from scratch rather than editing existing docs? If so, we'd better revamp the whole thing and go through the "battle card" preparation before writing full articles. Since I'm unclear about this, I'd like to have it clarified first and will stop my review here for now.

Reply:
I may be biased but operational purposes is quite straightforward to me. We need it to operate our platform and those 2 sentences have the same meanings. 
I am not talking about the customer use yet here, only that they can access it via grafana.

Now, maybe the why we use Loki instead of Grafana could be somewhere else, I wrote this under the after some discussions with @carillan81 and @pipo02mix .

The goal of this doc is to show what we have and show our expertise but maybe the battlecard approach was missed.

Regarding the structure, I put this under vintage for now because of the full revamp being done and I have no issues moving it and refining the content later on with said battlecard, but in the mean time, I think this type of doc is useful and needs to be there. I could be wrong also but it's still better than having nothing at the moment at least

Reply:
Agreed – I was writing my direct thoughts to challenge and make everyone think if everything still fits what we have here. We should check from scratch which pieces we put where, and if we can make it all shorter to increase chances of reading these articles. For example, does it fit into introductional, instructional or rather advanced parts of our docs, etc.?

Reply:
I'm always happy with any feedback :)